### PR TITLE
Add gofmt, goimports, vet, and codecov

### PR DESCRIPTION
**Pull request by saicheems**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,9 @@
-GAX: Google API eXtension
-=========================
+Google API Extensions for Go
+============================
 
-GAX provides the utilities for gRPC-based API client libraries.
+[![Build Status](https://travis-ci.org/googleapis/gax-ruby.svg?branch=master)](https://travis-ci.org/googleapis/gax-golang)
```
Comment (tbetbetbe) [inline README.md]: s/ruby/golang/

File: README.md [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+[![Code Coverage](https://img.shields.io/codecov/c/github/googleapis/gax-ruby.svg)](https://codecov.io/github/googleapis/gax-golang)
```
Comment (tbetbetbe) [inline README.md]: s/ruby/golang/

Comment (saicheems) [inline README.md]: Done

**Comment by tbetbetbe** [comment]
any idea what's causing the coverage failures ?

**Comment by saicheems** [comment]
I'm not sure... I thought it was that there wasn't already a coverage report for the base repo

**Comment by saicheems** [comment]
I suspect this is actually fine... I'm going to push and fix codecov in a follow up if there's actually a problem.

**Comment by saicheems** [comment]
Yeah, it's fixed now

**Pull request merged**